### PR TITLE
bugfix: allow multi-digit patch versions with apt repo

### PR DESCRIPTION
**Pull request by DarkGigaByte**

### CODE REVIEW

File: manifests/repo/apt.pp [hunk 1]
```diff
@@ -44,7 +44,7 @@
   } else {
     if $version =~ /^\d\.\d$/ {
       $repo_ver = "${version}/LATEST"
-    } elsif $version =~ /^(\d)\.(\d)\.(\d).*$/ {
+    } elsif $version =~ /^(\d)\.(\d)\.(\d).*?$/ {
```
Comment (igalic) [inline manifests/repo/apt.pp]: you only *capture* one digit here

File: manifests/repo/apt.pp [hunk 2]
```diff
@@ -44,7 +44,7 @@
...
       $repo_ver =  "${1}.${2}/${1}.${2}.${3}"
```
Comment (igalic) [inline manifests/repo/apt.pp]: are other digits not significant here?

**Comment by bastelfreak** [comment]
@DarkGigaByte thanks for the PR! A few things:
* Please rebase against latest master to get rid of the failures
* Please add spec tests to validate a few versions

**Comment by bastelfreak** [comment]
Hi. Thanks for raising this PR, but I'm going to close it due to inactivity. Please reopen it if you're still interested in getting this merged.

**Pull request closed without merging**